packages/eslint-plugin/src/hash.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
This should be its own package so it can be used in both codemod comment generation and linting